release: v1.49.0 (#5418)
apollographql/router: .changesets/fix_bryn_demand_control_formatting_hot_path.md

### Prevent formatting in hot path ([PR #5405](https://github.com/apollographql/router/pull/5405))

> This PR removed unneeded formatting in the hot path for demand control. Improving performance.

Reviewer: ```suggestion
Removes unneeded formatting in the hot path for demand control to improve performance.
```
